src/main/java/Duke.java

```diff
@@ -3,41 +3,43 @@
 public class Duke {
 
     public static void main(String[] args) {
-        System.out.println("Hello, I'm Duke!\n" + "How may I help you today?");
+        System.out.println("Hello, I'm Duke!\n" + "What would you like to do today?");
         Scanner response = new Scanner(System.in);
         String line = response.nextLine();
         int counter = 0;
```

Comment:
Perhaps could use a more meaningful variable name that is specific to the item that is being counted